packages/react/src/codemods/emotion-to-compiled/README.md

```diff
@@ -2,13 +2,30 @@
 
 > Codemod for easy migration from emotion.
 
+## Usage
+
+### Using `jscodeshift`
+
 ```bash
 # https://github.com/facebook/jscodeshift#usage-cli
 npx jscodeshift --parser=tsx --extensions=tsx --transform node_modules/@compiled/react/dist/codemods/emotion-to-compiled <path>
 
 # <path> is the source code folder eg. src
 ```
 
+### Using `@compiled/cli`
+
+```bash
+npx @compiled/cli --preset codemods
+# and follow the instructions
+```
+
+How instructions looks like (when run without `--preset codemods`. Please run it with `--preset codemods` for fast access):
+
+![emotion-to-compiled cli](./assets/emotion-to-compiled-cli.gif)
```

Comment:
can we update this please 

- new gif (or just remove for now)
- do we need to mention updating instructions? when i run `npx @compiled/cli` and then we release a new version, what happens?
- remove `caveats` section for now